certified-connectors/TrueDialog SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1762 @@
+            "CallbackToken": {
+              "type": "string",
+              "description": "CallbackToken"
+            },
+            "CallbackType": {
+              "type": "integer",
+              "format": "int32",
+              "description": "CallbackType",
+              "x-ms-visibility": "internal"
+            },
+            "CallbackURL": {
+              "type": "string",
+              "description": "CallbackURL",
+              "x-ms-visibility": "internal"
+            },
+            "TransactionId": {
+              "type": "string",
+              "description": "TransactionId"
+            },
+            "AccountId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "The ID of the account the callback is associated to",
+              "title": "AccountId"
+            },
+            "AccountName": {
+              "type": "string",
+              "description": "The name of the account the callback is associated to",
+              "title": "Account Name"
+            },
+            "CampaignId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "CampaignId"
+            },
+            "ChannelId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "ChannelId"
+            },
+            "ChannelCode": {
+              "type": "string",
+              "description": "Code of the channel the message has been sent over (the actual phone number for long codes)",
+              "title": "Channel Code"
+            },
+            "ContactId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "ID of the contact that submitted the keyword",
+              "title": "ContactId"
+            },
+            "Keyword": {
+              "type": "string",
+              "description": "The actual keyword that has been submitted",
+              "title": "Keyword"
+            },
+            "KeywordId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "ID of the keyword that has been submitted",
+              "title": "KeywordId"
+            },
+            "Message": {
+              "type": "string",
+              "description": "The actual message text",
+              "title": "Message"
+            },
+            "PhoneNumber": {
+              "type": "string",
+              "description": "The phone number of the end user that submitted the keyword",
+              "title": "PhoneNumber"
+            },
+            "SubscriptionId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "SubscriptionId",
+              "x-ms-visibility": "internal"
+            }
+          }
+        }
+      },
+      "put": {
+        "responses": {
+          "201": {
+            "description": "Keyword callback created"
```

Comment:
@TrueDialogDynamics Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
Added schema for all the responses in [this commit](https://github.com/microsoft/PowerPlatformConnectors/pull/1337/commits/d4d6d4c7b8aa28c0eceb87f9eccd4625ec7c5d84)